code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1049,11 +1038,7 @@
 	return
 
 /mob/living/carbon/human/get_species()
-
-	if(!species)
-		set_species()
-
-	return species.name
+	return dna.species.name

```

Comment:
maybe I overlooked you taking care of this, but be sure to look out for places which use `get_species` as a species identifier that might be passed to something like `set_species`

Reply:
I changed every single `set_species` to behave properly; this proc, AFAIK, "works", but it's not "ideal"

Ideally, we'll have a new proc that's something like `get_species_name` where a name is specifically needed, and everything else will use `is_species` (which will replaced the `if(get_species() == "Vulpkanin")` style checks.